examples/src/main/java/com/mapbox/navigation/examples/utils/extensions/Mappers.kt

```diff
@@ -8,7 +8,7 @@ import com.mapbox.navigation.base.route.model.Route
 
 fun Route.mapToDirectionsRoute(): DirectionsRoute {
     val duration = duration.toDouble()
-    val legs = legs?.legs?.let { it as List<RouteLeg> }
+    val legs = legs?.let { it as List<RouteLeg> }
```

Comment:
yep, it's trade-off, in next PR it's fixed. Made like this because there're no mappers. 
`val legs = legs?.legs?.let { it as List<RouteLeg> }`  is right way